microservices/biobuddy/biobuddy/api_models.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import dataclasses
+from typing import List, Optional, Dict, Any
+
+from openai.types.chat import ChatCompletionMessage
+
+from biobuddy.mixins import BaseModelMixin
+
+
+@dataclasses.dataclass
```

Comment:
Use pydantic.dataclasses, since dataclasses.dataclass would not be validated by fastapi and swagger, pydantic.dataclasses would